mne/viz/epochs.py

```diff
@@ -442,6 +438,249 @@ def plot_epochs(epochs, epoch_idx=None, picks=None, scalings=None,
+        pick_kwargs[t] = False
+    inds = np.concatenate(inds).astype(int)
+    if not len(inds) == len(picks):
+        raise RuntimeError('Some channels not classified. Please'
+                           ' check your picks')
+
+    # preallocation needed for min / max scaling
+    data = np.zeros((len(epochs.events), epochs.info['nchan'], len(times)))
+
+    for ii, epoch in enumerate(epoch_data):
+        pick = 0
+        for ind in inds:
+            data[ii, pick] = epoch[ind] / scalings[types[pick]]
+            pick += 1
+
+    # set up plotting
+    size = get_config('MNE_BROWSE_RAW_SIZE')
+    if size is not None:
+        size = size.split(',')
+        size = tuple(float(s) for s in size)
+    if fig_title is None:
+        fig_title = epochs.name
+        if epochs.name is None or len(fig_title) == 0:
+            fig_title = ''
+    fig = figure_nobar(facecolor='w', figsize=size)
+    ax = plt.subplot2grid((10, 15), (0, 0), colspan=14, rowspan=9)
+    ax.set_title(fig_title, fontsize=12)
+    ax.axis([0, duration, 0, 200])
+
+    ax_hscroll = plt.subplot2grid((10, 15), (9, 0), colspan=14)
+    ax_hscroll.get_yaxis().set_visible(False)
+    ax_hscroll.set_xlabel('Epochs')
+    ax_vscroll = plt.subplot2grid((10, 15), (0, 14), rowspan=9)
+    ax_vscroll.set_axis_off()
+
+    ax_vscroll.add_patch(mpl.patches.Rectangle((0, 0), 1, len(picks),
+                                               facecolor='w', zorder=2))
+    # populate vertical and horizontal scrollbars
+    for ci in range(len(picks)):
+        ax_vscroll.add_patch(mpl.patches.Rectangle((0, ci), 1, 1,
+                                                   facecolor=color[types[ci]],
+                                                   edgecolor=color[types[ci]],
+                                                   zorder=3))
+
+    vsel_patch = mpl.patches.Rectangle((0, 0), 1, n_channels, alpha=0.5,
+                                       edgecolor='w', facecolor='w', zorder=4)
+    ax_vscroll.add_patch(vsel_patch)
+
+    ax_vscroll.set_ylim(len(types), 0)
+    ax_vscroll.set_title('Ch.')
+
+    length = len(epochs.times)
+    typecolors = [colorConverter.to_rgba(color[c]) for c in types]
+    colors = list()
+    for ii in range(len(typecolors)):
+        colors.append(list())
+        for jj in range(len(epochs.events)):
+            colors[ii].append(typecolors[ii])
+    lines = list()
+    line = [(0, 0) for x in range(length)]
+
+    for channel in range(n_channels):
+        if len(colors) - 1 < channel:
+            break
+        lc = LineCollection(np.array([line] * len(epochs.events)),
+                            linewidths=0.5, colors=colors[channel], zorder=3)
+        ax.add_collection(lc)
+        lines.append(lc)
+
+    # concatenation
+    epoch_data = np.concatenate(data, axis=1)
+
+    ylim = [n_channels * 2.0 + 1, 0]
+    # make shells for plotting traces
+    offset = ylim[0] / n_channels
+    offsets = np.arange(n_channels) * offset + (offset / 2)
```

Comment:
Personally I would still allow the treillis plot but would default to False
and maybe remove at some point. This is new viz is much more fluid and
informative.